internal/rootcoord/timeticksync.go

```diff
@@ -101,20 +169,45 @@ func (t *timetickSync) UpdateTimeTick(in *internalpb.ChannelTimeTickMsg) error {
 	if !ok {
 		return fmt.Errorf("Skip ChannelTimeTickMsg from un-recognized proxy node %d", in.Base.SourceID)
 	}
+
+	// if from ddl operation, remove ddl timetick firstly
+	if ddl {
+		t.RemoveDdlTimeTick(in.DefaultTimestamp, reason)
+	} else {
+		// if ddl operation with smaller ts is in process, skip current ts update
+		ddlMinTs := t.GetDdlMinTimeTick()
```

Comment:
even if we remove current ddl, should we still check DdlMinTimeTick if we have multiple DDLs inqueue

Reply:
updated